src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
I think we want to allow multiple imports with the same name though. Otherwise you can't do something like import something qualified and with selective imports at the same time.